ui-grid/ui-grid.d.ts

```diff
@@ -1064,6 +1074,15 @@ declare module uiGrid {
         (scrollEvent: JQueryMouseEventObject): void;
     }
 
+    export interface sortChangedHandler {
+        /**
+         * Sort change event callback
+         * @param {IGridInstance} grid instance
+         * @param {IGridColumn} array of gridColumns that have sorting on them, sorted in priority order
+         */
+        (grid: IGridInstance, columns: IGridColumn[]): void;
```

Comment:
Thanks for your points, guys. I am new to github and I would like to edit this PR to incorporate review comments but don't know how to do so. I would like to keep only the sortChanged handler (generic one as suggested) and not include the undocumented gridHeight property.
Or is it better to close this PR and open a new one?